lib/index.js

```diff
@@ -247,23 +247,28 @@ api.createSignatureString = ({includeHeaders, requestOptions} = {}) => {
  * the request creating a signature & AuthzHeader.
  *
  * @param {Object} request - A request object.
- * @param {Object} options - Options for parsing such as clockSkew.
+ * @param {Object} [options={}] - Options for parsing such as clockSkew.
  *
  * @returns {Object} A parsed request with a signed header.
 */
 api.parseRequest = (request, options) => {
   assert.object(request, 'request');
+  // headers make up the major of coverted content used to create a signature
+  // so headers are required
   assert.object(request.headers, 'request.headers');
   if(options === undefined) {
     options = {};
   }
   if(options.headers === undefined) {
     options.headers = ['(created)'];
   }
+  // options are optional, but should always be an object
   assert.object(options, 'options');
+  // options.headers are also required, but is an array of covered content names
   assert.arrayOfString(options.headers, 'options.headers');
+  // clockSkew can't be infinite
   assert.optionalFinite(options.clockSkew, 'options.clockSkew');
```

Comment:
```suggestion
  // options are optional, but should always be an object
  assert.object(options, 'options');
  // options.headers are also required, but is an array of covered content names
  assert.arrayOfString(options.headers, 'options.headers');
  // clockSkew can't be infinite
  assert.optionalFinite(options.clockSkew, 'options.clockSkew');
```

Reply:
let me know if this addresses this: https://github.com/digitalbazaar/http-signature-header/pull/24/commits/8dc1062bfb161d3ca74fdf8318140d8a0676dbfa